user_manual/files/webgui/sharing.rst

```diff
@@ -154,6 +154,9 @@ change     Allows uploading a new version of a shared file and replacing it
 delete     Allows the users you share with to delete shared files
 ========== ===================================================================
    
+
+.. _create_upload_only_link_share_label:
```

Comment:
I think we need some more consistency here. Either we call the feature 'File drop' / 'Drop folder' or 'Upload-only link shares'. IMO we should use the 'drop' terms as they better reflect the use case. Upload-only does not imply that users can't see the file listing.